webserver/script/CostBenefitAnalysis/UploadToPostgres.py

```diff
@@ -0,0 +1,398 @@
+import boto3
+import json
+import csv
+import psycopg2
+from django.conf import settings
+
+
+class UploadToPostgres():
+    def __init__(
+        self,
+        load_profile
+    ): 
+        with open(settings.BASE_DIR[:-3] + 'script/CostBenefitAnalysis/postgres_info.json') as json_file:
+            postgres_info = json.load(json_file)
+        
+        # load_profile related
+        self.load_profile_year_len = 0
+        self.load_profile_result_dict = {}
+        self.load_profile_start_year = 0
+        self.load_profile = load_profile
+        print('self', self.load_profile)
```

Comment:
Can we remove this?

Reply:
thanks for the catch - will do, and I'll do a sweep check of print statements to make sure.